src/pages/edit_resource/contents_step/AdditionalFilesUpload.tsx

```diff
@@ -20,6 +20,7 @@ import ErrorBanner from '../../../components/ErrorBanner';
 interface AdditionalFilesUploadProps {
   additionalFileUploadUppy: Uppy;
   newContent: Content | undefined;
+  thumbnailUppy: Uppy;
```

Comment:
er denne tatt med for fremtidig bruk ? jeg finner ikke bruk av den

Reply:
Fjernet, var med fordi tidligere så ble miniatyrbilde contentet også vist fram her. Nå filtreres miniatyrbildene bort